external/kafka/src/main/scala/org/apache/spark/streaming/kafka/KafkaUtils.scala

```diff
@@ -684,3 +679,53 @@ private class KafkaUtilsPythonHelper {
     kafkaRDD.offsetRanges.toSeq
   }
 }
+
+private object KafkaUtilsPythonHelper {
+  private var initialized = false
+
+  def initialize(): Unit = {
```

Comment:
Why not directly call `SerDeUtil.initialize()` and `new PythonMessageAndMetadataPickler().register()` in the static codes of `KafkaUtilsPythonHelper`?